#167276162 Create inputForm Field molecule

akinyeleolat opened the pull request:
#### What does this PR do?

- This PR sets up an Input Form field molecule with styleguide

#### Description of task to be completed?

- setup input form field molecule
- setup styleguidist for the input form field molecule
- write tests for the input form field molecule
- refactor divider to display inline
- add web pack molecule module resolver
- add jest config molecule module resolver
- add molecule sections to style guide configuration

#### How should this be manually tested?

  - $ git clone https://github.com/andela/king-kong-ah-frontend.git
  - $ npm install
  - $ npm test

#### Any background context you want to provide?

- You would need npm to run this application

#### What are the relevant pivotal tracker stories?

- PT Story link - [#167276162](https://www.pivotaltracker.com/story/show/167276162)

#### Screenshots (if appropriate)
<img width="1432" alt="Screenshot 2019-07-13 at 7 25 18 PM" src="https://user-images.githubusercontent.com/19696366/61175259-fe86a500-a5a3-11e9-9c60-ec8edf155737.png">

benfluleck commented on src/components/UI/molecules/InputFormField/InputFormField.jsx:
```diff
@@ -0,0 +1,76 @@
+  label,
+  display,
+}) => {
+  return (
```
There should be a space here above the `return`

benfluleck commented on src/components/UI/molecules/InputFormField/InputFormField.jsx:
```diff
@@ -0,0 +1,76 @@
+        error={errorMessage}
+        onChange={onChange} />
+      {errorMessage
+        && <span><Text color='error' fontSize='small' display='block'>
```
I expect the color to be red not error

benfluleck commented on src/components/UI/molecules/InputFormField/InputFormField.jsx:
```diff
@@ -0,0 +1,76 @@
+  return (
+
+    <FlexContainer display={display}>
+      <label htmlFor={name}>
```
Do a Label molecule. Doing this here means any time the label is edited we need to edit here.

benfluleck commented on src/components/UI/molecules/InputFormField/InputFormField.spec.js:
```diff
@@ -0,0 +1,37 @@
+
+  it('should render a input password field component', () => {
+    const { container } = setUp({ type: 'password' });
+    expect(container).toBeTruthy();
```
This is not the correct test I expect a test for one of the props

benfluleck commented on src/components/UI/molecules/InputFormField/InputFormField.spec.js:
```diff
@@ -0,0 +1,37 @@
+
+
+  it('should render a input field component with error message', () => {
+    const { container } = setUp({ errorMessage: 'first name is required' });
```
This is not the correct test I expect a test for one of the props

Commit:
feat(Inputfield): create inputfield molecule

- Implement input field molecule
- Write test
- Create style guide

[Finishes #167276162]
